api/api/handlers.go

```diff
@@ -212,9 +216,10 @@ func CreatePayloadHandler(c *gin.Context) {
 				concat,
 			)
 		} else if body["type"] == "wkeys" {
-			commandString = fmt.Sprintf( // Borrowed from https://github.com/s-christian/pwnts/blob/master/site/site.go#L175
-
-				"CGO_ENABLED=0 go build -trimpath -ldflags \"-s -w -X main.ServerIP=%s -X main.ServerPort=%s -X main.CallbackFrequency=%s -X main.CallbackJitter=%s -X main.RootCert=%s\" -o agents/%s client/kclient/kclient.go",
+			commandString = fmt.Sprintf(
+				"docker run --rm -v %s:/build -w /build golang:1.22.3 "+
```

Comment:
I wonder if you can double escape docker to get to the host with this. Regardless, it's still an extra layer. Noice

Reply:
could prob break out into the build container, which only lasts a few seconds. With the recent UI changes with auth, im not too worried